src/components/app/Initializing.js

```diff
@@ -2,10 +2,18 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+/* @flow */
```

Comment:
Would you mind changing this to `// @flow` since that's what we've done on every other file? That'll make it easier to search if we have flow coverage in a file or not. Thanks!